### PR TITLE
H-2224, H-2239, H-2373: Bug fixes in entities table, redirect after signin

### PR DESCRIPTION
**Pull request by CiaranMn**
## 🌟 What is the purpose of this PR?

A few small bug fixes:
1. The entities table didn't switch back to non-file view when switching to a non-file type from a file type (via the sidebar)
2. The table wasn't sorting on properties, because the sort function tries to do `row[value]` but the properties were under `row.properties[value]`
3. The `return_to` if a user visited a page they needed to log in for wasn't taking account of dynamic paths (e.g. `/@[shortname]`

### 🚢 Has this modified a publishable library?

This PR:

- [x] does not modify any publishable blocks or libraries, or modifications do not need publishing

### 📜 Does this require a change to the docs?

The changes in this PR:

- [x] are internal and do not require a docs change

### CODE REVIEW

**Review by nonparibus — APPROVED** [review]

File: apps/hash-graph/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "exe": "tsx",
     "fix:clippy": "just clippy --fix",
     "lint:clippy": "just clippy",
+    "migrate": "cargo run --bin hash-graph --all-features -- migrate --user postgres --password postgres",
```
Comment (benwerner01) [inline apps/hash-graph/package.json:25]: I assume this is an intentional drive-by change

**Review by benwerner01 — APPROVED** [review]
Thanks Ciaran, the changes look good and are working for me 👍
